### ISSUE TITLE
add fuzzymatching

### ISSUE THREAD

**Issue by pavertomato** [OHIF/Viewers#707]
fuzzymatching  is part of qido standard https://www.dicomstandard.org/dicomweb/query-qido-rs/ but it's not present in ohif-core https://github.com/OHIF/ohif-core/blob/master/src/studies/services/qido/studies.js#L37 
but may be not all DICOMweb server support it
my idea is to add this to ohif-core and parameter to config file:
- if parameter is enabled UI will show checkbox to pass true or false as value of fuzzymatching to request
- if parameter is disabled UI won't contain checkbox and and won't pass fuzzymatching parameter to request.

**Comment by pavertomato** [comment]
any doubts about this plan?

**Comment by dannyrb** [comment]
@pavertomato, my only hold-up is:

> UI will show checkbox to pass true or false as value of fuzzymatching

In my opinion, if fuzzymatching is supported, it's almost always preferred unless we're encountering performance issues from very large datasets. Heck, I even wish we had a _single_ search field that fuzzymatched across all searchable fields, with an "advanced" search option that expanded to show more boxes if you needed fine grained filtering.

tl;dr

I like adding `supportsFuzzyMatching` to the server settings for the application. If true, I think we should use fuzzy matching by default. I don't think we need to add a checkbox to the UI. @swederik may wish to weigh in.
